LSST Catalog Access Tutorial.ipynb

```diff
@@ -48,13 +48,11 @@
    "source": [
     "import pandas\n",
     "from IPython.display import Markdown as md\n",
-    "try:\n",
-    "    from jupyterlabutils.notebook import get_catalog, retrieve_query\n",
-    "except ModuleNotFoundError:\n",
-    "    from rubin_jupyter_utils.lab.notebook import get_catalog, retrieve_query\n",
+    "\n",
```

Comment:
I need to update this soon.  Those packages are vestiges of nublado v1 and they need to be replaced with something better.